src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -815,6 +815,16 @@ def aks_create(cmd,
                dns_zone_resource_id=None,
                enable_custom_ca_trust=False,
                yes=False):
+    if disable_disk_driver or disable_file_driver:
+        msg = "Please make sure there are no existing PVs and PVCs that are used by the CSI driver before disabling."
+        if not yes and not prompt_y_n(msg, default="n"):
+            return None
+    if disable_snapshot_controller:
+        msg = "Please make sure there are no existing VolumeSnapshots, VolumeSnapshotClasses and VolumeSnapshotContents " \
+              "that are used by the snapshot controller before disabling."
+        if not yes and not prompt_y_n(msg, default="n"):
+            return None
+
```

Comment:
Please put these statements below `raw_parameters = locals()`. Also put them into `decorator.py`? It's hard to test the entry function (`aks_create`).

Reply:
done